lib/services/discourse.ts

```diff
@@ -104,14 +105,19 @@ export class DiscourseService extends Messenger implements ServiceListener, Serv
 			return new Promise<DiscourseEmitContext>((resolve) => {
 				resolve({
 					endpoint: {
-						api_key: data.toIds.token,
-						api_username: data.toIds.user,
+						method: 'POST',
+						qs: {
+							api_key: data.toIds.token,
+							api_username: data.toIds.user,
+						},
+						url: `https://${this.data.instance}/posts`,
 					},
 					payload: {
-						category: data.toIds.flow,
-						raw: `${data.text}\n\n---\n${Messenger.stringifyMetadata(data)}`,
+						'category': data.toIds.flow,
```

Comment:
Am changing tslint.json https://palantir.github.io/tslint/rules/object-literal-key-quotes/ to [true, "as-needed"] rather than [true, "consistent-as-needed"] okay?